.github/workflows/ci_windows.yml

```
@@ -16,7 +16,9 @@ on:
 jobs:
   gcc8-build:
     name: GCC8 build
-    if: github.event.review.state == 'approved'
+    # Only the third condition, this job is not executed because the condition is always false
+    # if this job is triggered by 'push' or workflow_dispatch.
```

ここのコメントがちょっと分かりづらかったので、表現を変更した方が良いと思いました。
```suggestion
    # For jobs triggered by pull_request_review, build task should run only if is in `approved` state.
```

---

こっちのほうが明確ですね．変更しました．